storage/api/Storage.Samples.Tests/CreateBucketWithTurboReplicationTest.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright 2021 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+using Xunit;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+[Collection(nameof(StorageFixture))]
+public class CreateBucketWithTurboReplicationTest
+{
+    private readonly StorageFixture _fixture;
+
+    public CreateBucketWithTurboReplicationTest(StorageFixture fixture)
+    {
+        _fixture = fixture;
+    }
+
+    [Fact]
+    public void CreateBucketWithTurboReplication()
+    {
+        CreateBucketWithTurboReplicationSample createBucketWithTurboReplication = new CreateBucketWithTurboReplicationSample();
+        var bucketName = Guid.NewGuid().ToString();
```

Comment:
Add a method on the fixture, something like GenerateBucketName, and add that bucket name into the TempBuckets (see the method CreateBucket in the fixture), so this bucket will be removed on clean up.